<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>736</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>922</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CNV Result Sheet Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="5">
     <widget class="QLineEdit" name="input_field">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Search Term:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ClinVar Data Recommendation ( &gt;= 80 ):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="generateButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="4" colspan="4">
     <widget class="QTextEdit" name="output_area">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="freqCombo">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Previous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Obligate (100%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Very frequent (99-80%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Frequent (79-30%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Occasional (29-5%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Very rare (&lt;4-1%)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Excluded (0%)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="fileOpenButton">
      <property name="maximumSize">
       <size>
        <width>108</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="fileBrowse">
      <property name="maximumSize">
       <size>
        <width>108</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QTextBrowser" name="recommendBrowser">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QTextBrowser" name="dictBrowser">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="fileName">
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ouput Editor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Orpha.Freq:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Coded by R&amp;D AIM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>fileBrowse</tabstop>
  <tabstop>freqCombo</tabstop>
  <tabstop>fileOpenButton</tabstop>
  <tabstop>output_area</tabstop>
  <tabstop>recommendBrowser</tabstop>
  <tabstop>input_field</tabstop>
  <tabstop>dictBrowser</tabstop>
  <tabstop>generateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
